src/autocomplete.js

```diff
@@ -57,7 +57,8 @@ export default class Autocomplete extends Component {
     required: false,
     tNoResults: () => 'No results found',
     dropdownArrow: DropdownArrowDown,
-    isQueryAnOption: (query, options) => options.map(entry => entry.toLowerCase()).indexOf(query.toLowerCase()) !== -1
+    isQueryAnOption: (query, options) => options.map(entry => entry.toLowerCase()).indexOf(query.toLowerCase()) !== -1,
+    isIosDevice: isIosDeviceDetected
```

Comment:
Again, it'd be good to avoid making this a prop just for tests - can we explore e.g. stubbing `navigator.userAgent` instead?

Reply:
Sure :)

Reply:
Changed as recommended. 
Have also moved the `isQueryAnOption` util function out of props (that one for a different reason although it equally didn't belong - now catering for the user defined suggestion template scenario, prompted by your other comment below..)